tensorrt_llm/_torch/models/modeling_cohere2_vision.py

```
@@ -0,0 +1,374 @@
+import os
+import dataclasses
+import copy
+from typing import Optional, List, Tuple, Literal
+
+import torch
+from torch import nn
+from ..._utils import nvtx_range
+from transformers import (AutoProcessor, AutoTokenizer, Cohere2VisionConfig, PreTrainedModel, PretrainedConfig)
+from transformers.activations import ACT2FN
```

Flake8 (7.3.0)

[error] 'transformers.activations.ACT2FN' imported but unused

(F401)